kube_apiserver_metrics/tests/test_kube_apiserver_metrics.py

```diff
@@ -108,23 +116,30 @@ def test_bearer(self):
         os.remove(temp_bearer_file)
         assert configured_instance["_bearer_token"] == APISERVER_INSTANCE_BEARER_TOKEN
 
-    def test_default_config(self, aggregator):
+    def test_default_config(self, aggregator, mock_read_bearer_token):
         """
         Testing the default configuration.
         """
         check = KubeAPIServerMetricsCheck('kube_apiserver_metrics', {}, {}, [minimal_instance])
-        apiserver_instance = check._create_kube_apiserver_metrics_instance(minimal_instance)
+
+        check.process = mock.MagicMock()
+        check.check(minimal_instance)
+
+        apiserver_instance = check.kube_apiserver_config
```

Comment:
Note to reviewers: testing `check.kube_apiserver_config` is more reliable since that's the scrapper config that will be actually used.